subprojects/catch2.wrap

```diff
@@ -0,0 +1,9 @@
+[wrap-git]
```

Comment:
Not sure if we want a git wrap or a tarball wrap :woman_shrugging:

Reply:
Done.